app/src/main/java/com/example/bigowlapp/SearchContactsToSupervise.java

```diff
@@ -89,6 +98,35 @@ private void loadContacts() {
         listContactsView = findViewById(R.id.listContacts);
         listContactsView.setAdapter(adapter);
 
+        //Check if users already has the app
+        listContactsView.setOnItemClickListener(new AdapterView.OnItemClickListener() {
+            // argument position gives the index of item which is clicked
+            public void onItemClick(AdapterView<?> arg0, View v, int position, long arg3)
+            {
```

Comment:
please reformat this file to use same style as everyone else.

You can do this in Android studio using the command `CTRL+ALT+SHIFT+L`, and selecting reformat code